tools/federation/src/main/java/org/eclipse/rdf4j/federated/QueryManager.java

```diff
@@ -75,7 +75,7 @@ public QueryManager(FederationManager federationManager, Repository repo) {
 	public void init() {
 
 		// initialize prefix declarations, if any
-		String prefixFile = Config.getConfig().getPrefixDeclarations();
+		String prefixFile = federationManager.federationContext.getConfig().getPrefixDeclarations();
```

Comment:
For consistency's sake I'd prefer the use of a getter method for `federationContext`. Can we make the field private and provide an accessor method instead?

Reply:
Done. Also addressed this in a few other places. See last commit